package.json

```diff
@@ -26,22 +26,22 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "next": "^13.0.0",
+    "next": "^13.2.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@types/node": "^16.18.2",
-    "@types/react": "^18.0.24",
-    "@typescript-eslint/eslint-plugin": "^5.41.0",
-    "@typescript-eslint/parser": "^5.41.0",
-    "eslint": "^8.26.0",
-    "eslint-config-next": "^13.0.0",
-    "eslint-config-prettier": "^8.5.0",
+    "@types/node": "^18.15.5",
```

Comment:
If you leave the node engine recommendation at 16 it is probably better to use node 16 types

Reply:
indeed, didn't notice that at first, thx!